entente/landmarks/landmark_composite_recipe.py

```diff
@@ -130,31 +124,27 @@ def original_and_reprojected_landmarks(self):
         for example in self.examples:
             example_id = example["id"]
             reprojected = self.reprojected_landmarks[example_id]
-            points = {}
-            for name in self.landmark_names:
-                reprojected_point = point_for_landmark_name(reprojected, name)
-                points[name] = {
-                    "original": example[name],
-                    "reprojected": reprojected_point,
+            result[example_id] = {
+                k: {
+                    "original": example[k],
+                    "reprojected": reprojected[k].tolist(),
```

Comment:
OK - I took a crack at revising this, to more consistently use numpy arrays in memory, and save the serializing until the `to_json` step.  It's more consistent, but also feels much more verbose.  Thoughts?

Reply:
I think types-wise, it's better, though I agree it's really verbose.

Is the rounding all to the same number of places? Probably it would be better to use a JSON serialization helper function which rounds and converts to lists all in one. `baiji.serialization` works like this, though since it depends on boto3 I've been trying to avoid it.

I think it is also okay to leave this as technical debt.